src/key/helper.js

```diff
@@ -348,6 +349,8 @@ export function sanitizeKeyOptions(options, subkeyDefaults = {}) {
     } else {
       options.algorithm = enums.publicKey.ecdh;
     }
+  } else if (options.symmetric) {
+    options.algorithm = enums.publicKey.aead;
```

Comment:
Yeah, I didn't give this a huge amount of thought, I just wanted something that worked to enable me to work on the other things. 

Key generation lets the user set the curve/key size respectively, so I just went with something similar.

Reply:
Also, key generation with `openpgp.generateKey`  is something I implemented but didn't commit as the main key is used for signatures which made it fail when it got to that (as we discussed a while back). I accounted for those defaults at that stage by changing: 
```
curve = rsaBits ? "" : curve;
```
to
```
  symmetric = rsaBits ? "" : symmetric;
  curve = rsaBits || symmetric ? "" : curve;
```

Reply:
Hmhm. I also think we should remove that `curve = rsaBits ? "" : curve` at some point in the future in favor of swapping the cases here, but yeah, since that's existing code I guess I can't really complain :)

Re. `openpgp.generateKey`, you could test it with `openpgp.generateKey({ subkeys: [{ symmetric: ... }] })`